src/containers/shared/components/Transaction/AMMDelete/test/AMMDeleteDescription.test.tsx

```diff
@@ -0,0 +1,17 @@
+import i18n from '../../../../../../i18n/testConfigEnglish'
+import mockAMMDelete from './mock_data/AMMDelete.json'
+import { Description } from '../Description'
+import { createDescriptionWrapperFactory } from '../../test'
+
+const createWrapper = createDescriptionWrapperFactory(Description, i18n)
+
+describe('AMMDelete: Description', () => {
+  it('renders description for AMMDelete transaction', () => {
+    const wrapper = createWrapper(mockAMMDelete)
+
+    expect(wrapper.html()).toBe(
```

Comment:
This is not a good way to test - it's difficult to edit the tests when things change minorly.

Reply:
Do you happen to know what might be better?

I think it's good to test that the translations work, and that the hyperlinks are added, since those are things that can't easily be checked via code review. 

And in this case, as far as I can tell, the only tangential change which would accidentally fail this test is if we changed how we hyperlink currencies (`<span class="currency">...</span>` and `<a class="" href="/token/FOO.rm5c42Crqpdch5fbuCdHmSMV1wrL9arV9">...</a>`)

The rest of the object is specific to the `AMMDelete` logic. 

(Or is the problem that I'm using `wrapper.html()` instead of a more specific lookup?)

Reply:
Nevermind - updated to have individual checks instead of a one-liner